web/app/index.html

```diff
@@ -39,6 +40,8 @@
     <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
     <script src="bower_components/angular-uuid/uuid.min.js"></script>
+    <script src="bower_components/leaflet/dist/leaflet.js"></script>
+    <script src="bower_components/leaflet/dist/leaflet-src.js"></script>
```

Comment:
These scripts are added automatically by the grunt wiredep task. You'll need to add an entry to the wiredep task to exclude one of the files.

Example: https://github.com/azavea/lgbs-dtax-mapping/blob/develop/web/Gruntfile.js#L182